charts/openstack-actions-runner/templates/secret.yaml

```
@@ -3,9 +3,7 @@ kind: Secret
 metadata:
   name: {{ .Release.Name }}-auth
 data:
-  githubToken:   {{ .Values.githubToken | b64enc }}
-  githubOrganization:   {{ .Values.githubOrganization | b64enc }}
-  cloudNineTenant: {{ .Values.cloudNineTenant | b64enc }}
-  cloudNineToken:  {{ .Values.cloudNineToken | b64enc }}
-  cloudNineUsername:  {{ .Values.cloudNineUsername | b64enc }}
-  cloudNinePassword:  {{ .Values.cloudNinePassword | b64enc }}
+  GITHUB_TOKEN:   {{ .Values.githubToken | b64enc }}
+  CLOUD_NINE_TOKEN:  {{ .Values.cloudNineToken | b64enc }}
+  CLOUD_NINE_USERNAME:  {{ .Values.cloudNineUsername | b64enc }}
+  CLOUD_NINE_PASSWORD:  {{ .Values.cloudNinePassword | b64enc }}
```

just add a `\n` here
```suggestion
  CLOUD_NINE_PASSWORD:  {{ .Values.cloudNinePassword | b64enc }}

```